docs/metrics.md

```diff
@@ -69,6 +114,26 @@ Here's an example of a complete telemetry ping:
 }
 ```
 
+A Redshift schema for the payload:
+
+```js
+local schema = {
+--   column name                   field type   length  attributes   field name
+    {"timestamp",                  "TIMESTAMP", nil,    "SORTKEY",   "Timestamp"},
```

Comment:
How high resolution is this timestamp? How does redshift handle collisions on the timestamp field if there are collisions?